### PR TITLE
Westmidlands-class 4-farzad-nazif- JavaScript-Core-1-Coursework-Week1

### PR DESCRIPTION
**Pull request by farzad-nazif**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Farzad Nazif
- Your City: Westmidlands - Birmingham
- Your Slack Name: Farzad-nazif

# Homework Details

- Module: JavaScript-Core-1
- Week: 1

# Notes

- What did you find easy?
understanding the var and functions
- What did you find hard?
the last challenge of extra folder
- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: extra/1-currency-conversion.js [hunk 1]
```diff
@@ -15,7 +18,11 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  let priceToBeConverted = 0.99*price;
+  let convertedPrice = priceToBeConverted * 5.7;
+  return parseFloat(convertedPrice.toFixed(2));
```
Comment (JanefrancessC) [inline extra/1-currency-conversion.js:24]: Nice job with the parseFloat(). I just got to know about it from reviewing your codes... Great job Farzad!

File: mandatory/3-function-output.js [hunk 2]
```diff
@@ -2,15 +2,20 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
+// Comment : math.random gives us a random number between 0 and 1. then we times it by 10, we will have a random number between 0 and 10.
```
Comment (JanefrancessC) [inline mandatory/3-function-output.js:5]: From my findings on Google, I could be wrong though, the getRandomNumber function will generate only floating numbers between 1 - 10

File: mandatory/4-tax.js [hunk 3]
```diff
@@ -17,7 +20,10 @@ function calculateSalesTax() {}
...
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(productPrice) {
+  let total = calculateSalesTax(productPrice);
+  return "£" + total.toFixed(2);
+}
 
 /* 
 ===================================================
```
Comment (JanefrancessC) [inline mandatory/4-tax.js:29]: Had a great time going through your codes. it gave me alternate perspectives to solve a problem! 

Thumbs up Farzad!

**Review by JanefrancessC — COMMENTED** [review]
Fantastic job!

File: mandatory/3-function-output.js (hunk 2 above)
Comment (farzad-nazif) [reply mandatory/3-function-output.js:5]: math.random() gives you a number between 0 and 1 , if you times it by 10 , then gives you 0 to 10

File: mandatory/4-tax.js (hunk 3 above)
Comment (farzad-nazif) [reply mandatory/4-tax.js:29]: Thanks a lot Chioma

**Review by alicagatay — CHANGES_REQUESTED** [review]
Nice job Farzad, I really liked the way you solved some of these problems. Keep up the good work.

**Pull request closed without merging**